examples/java/src/main/java/com/xebia/functional/xef/java/auto/Animals.java

```diff
@@ -18,18 +44,20 @@ private static class Story {
         public Animal animal;
         public Invention invention;
         public String story;
```

Comment:
The same about the comment in the class Art with the field art, rename the field story